Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'bind-it'
 gem 'capistrano'
 gem "capistrano-db-tasks", require: false
 gem 'capistrano-rails'
+gem 'chartkick'
```

Comment:
You could use `highchart` javascript library, which has already been used in Expertiza.

Reply:
I mentioned using `highchart`

Reply:
Sorry, but the team had no prior knowledge on javascript and we had a time constraint as maximum time was invested in setting up the dev environment.
We had mentioned in our early design that we are keen to use chartkick for graphs.